docs/REST/authentication.md

```diff
@@ -407,3 +407,82 @@ func getHmac(content string, key []byte) string {
 	return base64.StdEncoding.EncodeToString(hmac.Sum(nil))
 }
 ```
+### Python
+```python
+
+import base64
+import hashlib
+import hmac
+from datetime import datetime
+import six
+
+def sign_request(method, url, body, connection_string):
```

Comment:
>connection_string [](start = 36, length = 17)

I think we need to do a little modification here. For other languages, the input is credential and secret instead of connection string. So in our case, we don't need the parse connection string helper method.